lib/get/onValue.js

```diff
@@ -58,7 +58,10 @@ module.exports = function onValue(model, node, seed, depth, outerResults,
         }
     }
 
-    else {
+    else if (node && nodeType === undefined && nodeValue === undefined) {
```

Comment:
Required to ensure paths that terminate before a leaf (e.g. due to empty keysets) still include the key prior to the empty step.